web/templates/attendance/fill_course_attendance.html.eex

```diff
@@ -71,6 +71,56 @@
                               <i class="material-icons attendance-form__attendance attendance--absent">add_circle</i>
                               <i class="material-icons attendance-form__attendance attendance--not-filled">radio_button_unchecked</i>
                             </button>
+                            <%= hidden_input(fh, :comment) %>
+                            <button
+                              type="button"
+                              data-attendance-id="<%= fh.id %>"
+                              class="mdl-button mdl-js-button mdl-button--icon show-attendance-modal"
+                            >
+                              <i
+                                class="
+                                  material-icons
+                                  attendance-comment__button
+                                  <%=
+                                    if fh.data.comment && String.length(fh.data.comment) > 0 do
+                                      'attendance-comment__button--set'
```

Comment:
In Elixir, single-quoted words are char lists and double-quoted words are strings. You should use strings generally because char lists are usually used to interface with erlang. This might be a problem when using, for example, the `String` module.